chapter_optimization/adam_vn.md

```diff
@@ -94,14 +100,18 @@ The former works arguably slightly better in practice, hence the deviation from
 Typically we pick $\epsilon = 10^{-6}$ for a good trade-off between numerical stability and fidelity.
 -->
 
-*dịch đoạn phía trên*
+Không giống như RMSProp, bản cập nhật của chúng tôi sử dụng động lượng $\hat{\mathbf{v}}_t$ thay vì chính gradient.
+Hơn nữa, có một sự khác biệt nhỏ về mặt thẩm mỹ khi việc thay đổi kích thước xảy ra bằng cách sử dụng $\frac{1}{\sqrt{\hat{\mathbf{s}}_t} + \epsilon}$ instead of $\frac{1}{\sqrt{\hat{\mathbf{s}}_t + \epsilon}}$.
```

Comment:
```suggestion
Hơn nữa, có một sự khác biệt nhỏ về mặt thẩm mỹ khi thực hiện phép chuyển đổi bằng cách sử dụng $\frac{1}{\sqrt{\hat{\mathbf{s}}_t} + \epsilon}$ instead of $\frac{1}{\sqrt{\hat{\mathbf{s}}_t + \epsilon}}$.
```